### DISCUSSION TITLE
Maybe we could have a meta issue for minifier and pin it

### DISCUSSION THREAD

**Discussion by Austaras** [swc-project/swc#2933] [General]
title

**Comment by kdy1** [comment]
Of course, but is there any reason for it?

**Reply by Austaras** [reply]
For tracking progression and maybe compare to other minifier. I'm recently considering switch to swc for minify but haven't found a reliable source for performance and bundlesize.

**Reply by kwonoj** [reply]
Makes sense to me too. And / Or, maybe we can have labels for each scope (transform? minify? bundle?) we can issue labels as well?

**Reply by kdy1** [reply]
> we can have labels for each scope (transform? minify? bundle?)

I think this would be great, thanks!

**Reply by aelbore** [reply]
@Austaras if i understand your question you want a a progress that compares to other minifier like esbuild, terser and etc?
if yes you can check this repository
https://github.com/privatenumber/minification-benchmarks

**Reply by pastelsky** [reply]
Is there a list of bugs/performance issues that are preventing SWC from recommending this from production usage on its site?
An umbrella issue might help people track production readiness.

**Reply by Austaras** [reply]
there is [one](https://github.com/swc-project/swc/issues/2934) alreay, just not much thing in there
